package/MDAnalysis/topology/tpr/utils.py

```diff
@@ -403,11 +406,15 @@ def do_mtop(data, fver):
     top.add_TopologyAttr(Impropers([improper for improper in impropers
                                     if improper]))
 
+    if any(elements):
```

Comment:
Is it worth throwing a warning here if the element list is imcomplete? We do something like that with the TOPParser: https://github.com/MDAnalysis/mdanalysis/blob/784ccc717dba2ceff7fa5233bc3cc9d4640c2464/package/MDAnalysis/topology/TOPParser.py#L291-L293

Reply:
I am not sure about that one. In normal operation, there should not be a warning. A virtual site in your water model or a coarse-grained models are normal operation. Also, the whole point of using a TPR is that there is nothing to guess.

Reply:
So in all honesty when I wrote that comment I somehow had convinced myself that there were cases where you would have negative atomnumber values when doing antechamber -> parmed -> gmx conversions. But looking back at a few gmx dump outputs I was (happily) mistaken.

If the negative numbers only commonly occurs in CG or EP situations then I'm happy going without warning.